mathesar_ui/src/systems/table-view/link-table/NewColumn.svelte

```diff
@@ -9,7 +9,7 @@
 
   import Pill from './LinkTablePill.svelte';
 
-  const label = 'Column Name';
+  export let label = 'Name of New Column';
```

Comment:
This string should be translated. I recognize that it wasn't translated before. But it would be good to make that change here. If there's a good reason for _not_ translating it, then we ought to add a code comment explaining why.